RST2-Programiranje-Vaje/Vaje_1.cs

```diff
@@ -65,23 +119,78 @@ public static int Naloga123(int mesec)
         /// </summary>
         public static void Naloga131()
         {
-            // Pridobimo trenutni čas
-            DateTime trenutniCas = DateTime.Now;
+            DateTime date1 = DateTime.Now;
+            
+            while((date1.Hour * date1.Minute * date1.Second % 17) != 0) {
+                Thread.Sleep(500);
+                date1 = DateTime.Now;
+            }
+            Console.WriteLine($"{date1} can be divided by 17, when hours, minutes and seconds are multiplied with each other.");
+        }
 
-            // Ponavljamo, dokler produkt ni deljiv s 17
-            while ((trenutniCas.Hour * trenutniCas.Minute * trenutniCas.Second) % 17 != 0)
-            {
-                // Uporabimo formatiranje v interpolaciji
-                Console.WriteLine($"{trenutniCas:HH:mm:ss}");
+        /// <summary>
+        ///Zapišite metodo, ki v danem seznamu realnih števil izbriˇše vse vrednosti,
+        ///ki so manjše od začetnega povprečja. Naredite dve implementaciji: eno z zanko for in eno
+        ///z zanko foreach
+        /// </summary>
+        
+        public static void Naloga132(ref List<float> myArr){
+            float sum = 0;
+            int l = myArr.Count;
+            foreach (float item in myArr) {
+                sum += item;
+            }
+            float avg = sum / l;
```

Comment:
nitpick: l is not a good variable name. Length would be better, or consider just dividing by myArr.Count as l is not used anywhere else.